### ISSUE TITLE
Transaction did not send

### ISSUE THREAD

**Issue by Julka30** [DigiByte-Core/digibyte#237]
**Expected behavior**
Attempted to send all DGB from digibyte core wallet to binance.

**Actual behavior**
The transaction did not send, screen display has a question mark to the left of date/time/label/amount.

**To reproduce**
I had closed the wallet right away after signed the transaction.

**System information**
I have Digibyte Core 8.22.0 rc-4 installed on Windows pc deskop. I was informed by Digibyte support, that I am not the first person, who has that problem, transaction should be send in the next couple of days. I hope it will happen, will let you know.

**Comment by Julka30** [comment]
The transaction was not sent. I was waiting 3 weeks. After the time I delete everything from disc ecxept wallet.dat and install from the beggining. It took 7 days to sync. That actually did not fix the problem. I was trying to find my private keys by console window to sweep to another wallet, but without the result. In the meantime somehow there showed an option on the transaction to abandon and I did that. The coins came back to my wallet and they were successfully send to the market.

**Comment by JaredTate** [comment]
Thanks for filing an issue and giving us feedback on RC4. I have seen issues like this with every version of BTC and DGB since v2. It can be affected by your local network/ internet/ ISP etc. However, a restart of the wallet always fixed it, or right-clicking abandon.

What kind of wallet was set up? Legacy, HD, descriptor? Was this an old import wallet.dat?  Did you use coin control to select inputs?

**Comment by Julka30** [comment]
Hi,
It was Digibyte core wallet on my laptop, which I do not use often. The
main problem was that there was not private key when I installed it in
2018. The wallet was opened maybe 4 times since that time. In 2022 I only
updated it to move coins and after a month sent them back. In april this
year when I installed the newest version I could not send the coins as
there was a command that the balance doesnt match the funds, no matter how
many coins I was trying to send. I was advised to upgrate it to rc4 and I
did and there showed another issue - the one decribed on github. The guys
from telegram support helped me a lot  and asked to fill it on github, so I
did.
After many tries of everything finally I was able to send coins to the
market.



pon., 20 maj 2024, 19:11 użytkownik Jared Tate ***@***.***>
napisał:
